META-INF/MANIFEST.MF

```diff
@@ -15,5 +15,6 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.lsp4j,
  org.eclipse.lsp4j.jsonrpc
 Automatic-Module-Name: LSP_CheckerFramework
-Bundle-RequiredExecutionEnvironment: JavaSE-12
```

Comment:
The changes here will not be added to the final PR. It is the environment I am currently working with...should not be submitted to the master branch.